packages/medusa/src/loaders/custom-api.ts

```diff
@@ -0,0 +1,56 @@
+import { Express, NextFunction, Request, Response, Router } from "express"
+import { ConfigModule } from "../types/global"
+
+enum AllowedMethods {
```

Comment:
Suggestion: I don't think we should block that and instead allow all verbs like ` 'all' | 'get' | 'post' | 'put' | 'delete' | 'patch' | 'options' | 'head'`